timers.ts

```diff
@@ -56,34 +57,28 @@ export function setTimeout(
   dispatch.sendMsg("timers", {
     command: pb.Msg.Command.TIMER_START,
     timerStartId: timer.id,
-    timerStartInterval: false,
+    timerStartInterval: interval,
     timerStartDuration: duration
   });
   return timer.id;
 }
 
-// TODO DRY with setTimeout
+export function setTimeout(
+  cb: TimerCallback,
+  duration: number,
+  // tslint:disable-next-line:no-any
+  ...args: any[]
+): number {
+  return setTimer(cb, duration, false, args);
+}
+
 export function setInterval(
   cb: TimerCallback,
   repeat: number,
   // tslint:disable-next-line:no-any
   ...args: any[]
 ): number {
-  const timer = {
-    id: nextTimerId++,
-    interval: true,
-    duration: repeat,
-    args,
-    cb
-  };
-  timers.set(timer.id, timer);
-  dispatch.sendMsg("timers", {
-    command: pb.Msg.Command.TIMER_START,
-    timerStartId: timer.id,
-    timerStartInterval: true,
-    timerStartDuration: repeat
-  });
-  return timer.id;
+  return setTimer(cb, duration, true, args);
```

Comment:
s/duration/repeat

Reply:
Why must be `repeat`? The MDN uses the word `duration` :(

Reply:
@yorkie I'm not a native speaker and don't know if there is any grammatical problem with that, but Node.js is using the name `repeat`

Try this on the command line:
```js
setTimeout + ""
```

Reply:
Sorry about that, MDN uses the word `delay`, I will follow your comments :)

Reply:
er - try `setInterval + ""`

Reply:
Actually you could see https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout, in MDN, it defines the 2nd argument's name to be `delay`, not the `repeat`.

Now let's just keep it with what we did before :)

Reply:
@yorkie so you must change line 77 as well, you previous failure was due to that : )

Reply:
@qti3e I have fixed it to use `repeat`, which would you prefer do, then I will follow it :)

Reply:
I personally like `delay` because that's what MDN uses and that's exactly what the definition is:

> The time, in milliseconds, the timer should **delay** before execution